iOS-#21 [Calculator View] Meterial Picker

tngusmiso opened the pull request:
재료 선택 Picker UI 구현
- 터치한 입력 버튼 종류에 따른 keyboard/picker 전환
- 입력 버튼의 글자를 picker에서 선택한 재료 명으로 변경
- 검색 버튼을 터치하여 재료 검색 뷰로 전환
- 검색화면에서 재료를 선택하면, 검색 화면 종료 후 입력 버튼의 글자를 선택한 재료 명으로 변경

tngusmiso committed:
iOS-#21 [계산기] 재료 선택 Picker 구현
- xib UIView 사용
- 터치하는 입력 버튼에 따라 숫자키보드, 재료선택 픽커 표시
- picker에서 선택한 재료로 입력 버튼의 내용 변경

tngusmiso committed:
iOS-#21 [계산기] 재료 검색 페이지로 이동

hihiboss commented on Amoogye/Calculator/CalculatorViewController.swift:
```diff
@@ -11,12 +11,47 @@ import UIKit
     var numericKeyboardButtonTitles = [String]()
 
+    // 입력 버튼
+    @IBOutlet weak var resPortionTextfield: UITextField!    // 변환 전 인분
```
res 가 어떤단어의 줄임말인가요??

tngusmiso replied:
source 생각하고 src한다는게 res라고 했네요..

hihiboss commented on Amoogye/Common/MeterialPicker/MeterialPickerView.swift:
```diff
@@ -0,0 +1,96 @@
+
+import UIKit
+
+class MeterialPickerView: UIView {
```
MeterialPickerView랑 PickerView랑 차이가 뭐야??

hihiboss replied:
MeterialPickerView랑 PickerView랑 파일 분리하는게 나을듯!

hihiboss commented on Amoogye/Common/MeterialSearchView/Cells/MeterialSearchResultCell.swift:
```diff
@@ -0,0 +1,32 @@
+        guard let selectedMeterialName = resultLabel.text else {
+            return
+        }
+//        delegate?.selectMeterial(name: selectedMeterialName)
```
이거 왜 주석처리??

hihiboss commented on Amoogye/Common/MeterialSearchView/MeterialSearchView.swift:
```diff
@@ -0,0 +1,77 @@
+}
+
+class SearchView: UIView {
+
```
여기도 클래스별로 파일 분리하는게 좋아보임!!

hihiboss reviewed (dismissed):
동사를 사용하는 네이밍은 동사 + 명사 형태가 좋음!!
(XXButtonClick -> clickXXButton)

hihiboss commented:
코멘트 단거 답글 달아주거나 반영하면 머지가자!!!

tngusmiso committed:
iOS-#21 [계산기] check reviews
- 동사+명사 형태의 네이밍으로 변경
- 오타 수정
- MeterialPickerView랑 PickerView 분리
- MeterialSearchView랑 SearchView 분리

hihiboss reviewed (approved):
LGTM :)
